rejson.go

```diff
@@ -3,30 +3,43 @@ package rejson
 import (
 	"encoding/json"
 	"fmt"
+	"strings"
 
 	"github.com/gomodule/redigo/redis"
 )
 
-// PopArrLast gives index of the last element for JSONArrPop
-const PopArrLast = -1
+const (
+	// PopArrLast gives index of the last element for JSONArrPop
+	PopArrLast = -1
+	// DebugMemorySubcommand & DebugHelpSubcommand provide the corresponding sub commands for JSONDebug
+	DebugMemorySubcommand = "MEMORY"
+	DebugHelpSubcommand   = "HELP"
```

Comment:
Can you add a comment for this const

Reply:
The comments for `DebugMemorySubcommand` & `DebugHelpSubcommand` are combined to a single comment.